packages/adblocker-benchmarks/run.js

```diff
@@ -80,10 +80,31 @@ function isSupportedUrl(url) {
   );
 }
 
+function looksLikeHostFilter(raw) {
+  // https://en.wikipedia.org/wiki/Hostname#Syntax
```

Comment:
@gorhill this is my idea of what a host filter is. I hope I've got it right! Let me know if you think I've missed something.